README.md

```diff
@@ -46,14 +46,27 @@
 
 # OpenTelemetry Collector Contrib
 
-This is a repository for OpenTelemetry Collector contributions that are not
-part of the [core
-distribution](https://github.com/open-telemetry/opentelemetry-collector) of the
-Collector. Typically, these contributions are vendor specific
-receivers/exporters and/or components that are only useful to a relatively
-small number of users.
-
-> Please note that this repository and its releases are a superset of the core repository.
+This is a repository for OpenTelemetry Collector components that are not suitable for the  [core repository](https://github.com/open-telemetry/opentelemetry-collector) of the collector. Some of these components are part of the core distribution, such as the Jaeger and Prometheus ones, but most of the components here are only available as part of the "contrib" distribution. Users of the OpenTelemetry Collector are also encouraged to build their own custom distributions with the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder), using the components they need from the core repository, the contrib repository, and possibly third-party or internal repositories.
+
+The official distributions, core and contrib, are available as part of the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository.
+
+Each component has its own stability and support levels, as defined in the following sections.
+
+## Stability levels
```

Comment:
What happens if a component is stable for a telemetry signal but not for others?  For example, how does the current status of the OTLP receiver defined [here](https://github.com/open-telemetry/opentelemetry-collector#status) map to these levels? I think it would not make sense to consider it stable unless logs functionality was under a feature flag.

Do we also want to make a distinction between configuration stability and behavior stability (say, do we want to allow expressing 'this component behavior is stable but the way it is configured may change in breaking ways')?

Reply:
We definitely need a dimension for the signal type, but I don't think it's a good user-experience to break it even more in config vs. behavior stability. From the user's perspective, they are all part of the same contract, IMO.

Reply:
I added a few words about it, would you mind taking a look?

Reply:
Looks good and agree that the distinction between configuration and behavior may not be a good user experience. I left a comment below but we can resolve this one!